lib/tasks/copy_vetted_documents.rake

```diff
@@ -0,0 +1,38 @@
+desc 'Copy all vetted documents from source tenant to destination tenant'
+task :copy_vetted_documents, [:source_tenant, :destination_tenant]  => :environment do |t, args|
+
+  # set tenant variables
+  source = args[:source_tenant]
+  destination = args[:destination_tenant]
+  starting_tenant = Apartment::Tenant.current
+
+  # switch to source tenant
+  Apartment::Tenant.switch(source)
+
+  # gather all vetted documents from source tenant
+  vetted_docs = Document.where(vetted: true).to_a
+
+  # switch to destination tenant
+  Apartment::Tenant.switch(destination)
+
+  # copy each vetted document from source tenant to destination
+  # tenant IF destination tenant does not already have a document
+  # with the same slug,
```

Comment:
If there's a doc with the same slug, we're skipping? Might be good to make a list of skipped docs, by slug.

Reply:
Good idea - added this 👍